implementations/rust/ockam/ockam_command/src/tcp/inlet/create.rs

```diff
@@ -215,8 +229,21 @@ impl CreateCommand {
                     &cmd.to
                         .to_string()
                         .color(OckamColor::PrimaryResource.color())
-                ),
-            )
+                )
+            } else {
+                fmt_ok!(
+                    "TCP Inlet {} on node {} will automatically connect\n",
```

Comment:
To keep the messages consistent, we should use `inlet` (in lowercase). For example: `The inlet {} on node {} will automatically connect`.

I would also rephrase this to `The inlet {} on node {} will automatically connect when the outlet becomes available`.